### ISSUE TITLE
letyrodri L02E12

### ISSUE THREAD

**Issue by letyrodridc** [mikeudacity/robo-pace-betatest#120]
In test run, this message appears above the images:

libdc1394 error: Failed to initialize libdc1394

**Comment by cangelogamboa** [comment]
Thanks for posting about this.  I fixed a big issue and made a little issue ... that's progress... right...?
I am looking into getting error free ASAP!

**Comment by letyrodridc** [comment]
Yes, the test code run and displays the three images. But above all that images there is that error message.
